scripts/verify_cairo_listings.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 set -e
 
-for d in ./docs/listings/*; do (cd "$d" && scarb test); done
+for d in ./docs/listings/*; do (cd "$d" && scarb check); done
```

Comment:
note: Instead of calling `scarb test` we should call `scarb check`, some tests in examples are failing on purpose.

Reply:
How did it work before?

Reply:
Wdym exactly 😅 ? Some packages have tests which fail on purpose, so the old `scarb test` would result in failure of this script, hope it clarifies.